### ISSUE TITLE
Does your script support image rotation prior to uploading files to firebase storage?

### ISSUE THREAD

**Issue by inglesuniversal** [InCuca/ic-firebase-uploader#5]
specifically iPhone images which look sideways due to the Exif data (6)

Regards

**Comment by klarkc** [comment]
No, but you can listen to [upload event](https://github.com/InCuca/ic-firebase-uploader/blob/master/src/ic-firebase-uploader/ic-firebase-uploader.vue#L172), [edit image file with canvas](https://stackoverflow.com/a/17412387) on event call and then call upload (passed in event payload).

**Comment by inglesuniversal** [comment]
I'll try that thanks
